docs/getting_started/data_owner_demo.md

```diff
@@ -9,6 +9,12 @@ hide:
 
 ## Overview
 
+The Benchmark owner provides only a small data sample for development purposes. However, the real model evaluations would be held on your side as a Data Owner. You have to prepare your own dataset that alignes with the benchmark's expected input format and submit information about it to the MedPerf server.
```

Comment:
```suggestion
As a data owner, you plan to run a benchmark on your own dataset. Using MedPerf, you will prepare your (raw) dataset and submit information about it to the MedPerf server. You may have to consult the benchmark committee to make sure that your raw dataset aligns with the benchmark's expected input format.
```
I'm afraid it was a bit confusing, what do you think now? Note also that I removed the part that mentions the benchmark's sample dataset because in reality I think a data owner wouldn't even know that such a topic exists.

Reply:
Yes, lgtm